packages/astro/src/core/config/schema.ts

```diff
@@ -311,6 +311,10 @@ export const AstroConfigSchema = z.object({
 							return langs;
 						})
 						.default([]),
+					langAlias: z
+						.record(z.string(), z.string())
+						.optional()
+						.default(ASTRO_CONFIG_DEFAULTS.markdown.shikiConfig.langAlias!),
```

Comment:
Why do we need ! here?

Reply:
I just copied what we do for the other options